src/main/java/org/opensearch/security/compliance/ComplianceConfig.java

```diff
@@ -148,6 +152,11 @@ private ComplianceConfig(
             try {
                 auditLogPattern = DateTimeFormat.forPattern(destinationIndex); // throws IllegalArgumentException if no pattern
             } catch (IllegalArgumentException e) {
+                log.warn(
```

Comment:
I'm unsure how users were suppose to troubleshoot errors in their format string without any error message - added this to aid in troubleshooting.